game/script-topics.rpy

```diff
@@ -1824,6 +1824,70 @@ label talk_sleeping_well:
 
     return
 
+label talk_natsukis_hairstyle:
+    if jn_affinity.get_affinity_state() >= jn_affinity.ENAMORED:
+        n "Hmm?{w=0.2} My hairstyle?"
+        n "Why do you ask,{w=0.1} [player]?{w=0.2} Looking for a stylist?"
+        n "Ehehe."
+
+    elif jn_affinity.get_affinity_state() >= jn_affinity.NORMAL:
+        n "Huh?{w=0.2} My hairstyle?"
+        n "Wait...{w=0.3} are you messing with me?{w=0.2} Was that an insult?"
+        n "You better not be teasing me,{w=0.1} [player]..."
+
+    else:
+        n "...Huh?{w=0.2} Oh.{w=0.2} My hair."
+        n "I'm...{w=0.3} surprised you care enough to ask about that."
+
+    n "Well,{w=0.1} anyway."
+    n "I never really thought about it that much,{w=0.1} honestly."
+    n "I just thought ponytails would look cute on me..."
+
+    if jn_affinity.get_affinity_state() >= jn_affinity.NORMAL:
+        n "...Yeah,{w=0.1} yeah.{w=0.2} I know what you're thinking,{w=0.1} [player]."
+
+        if jn_affinity.get_affinity_state() >= jn_affinity.ENAMORED:
+            n "Was I wrong...?"
+            n "Ehehe.{w=0.2} I thought not."
+
+    n "As for the bangs,{w=0.1} I...{w=0.3} always found it difficult to get my hair cut."
+
+    if jn_affinity.get_affinity_state() >= jn_affinity.NORMAL:
+        n "It just costs so much,{w=0.1} you know?{w=0.2} It's super dumb!"
+        n "Like...{w=0.3} I don't get it at all!"
+        n "And the annoying thing is that if I were a guy,{w=0.1} it'd be way cheaper!{w=0.2} What's up with that?"
+        n "Ugh...{w=0.3} anyway."
+
+    else:
+        n "I was always kinda short."
+        n "...And no,{w=0.1} {i}not{/i} in the physical sense."
+
+    n "As for my hairclip?{w=0.2} It's just to keep my hair out of my eyes."
+    n "Looking good is a bonus,{w=0.1} but I mostly just got tired of brushing my hair out of my face."
+    n "Have I thought about other hairstyles?{w=0.2} Well..."
+
+    if jn_affinity.get_affinity_state() >= jn_affinity.ENAMORED:
+        $ chosen_tease = random.choice(jn_globals.DEFAULT_PLAYER_TEASE_NAMES)
+        n "I'm pretty sure I already let my hair down around you,{w=0.1} [chosen_tease]."
+        n "Wouldn't you agree?"
```

Comment:
This corresponds to the line above; Natsuki is basically chaining off her play on words in a teasing way. I'll shift the line up, though.